### ISSUE TITLE
No status icons in Fedora 27/Wayland

### ISSUE THREAD

**Issue by techtonik** [nitroshare/nitroshare-desktop#159]
Nitroshare is completely invisible on Fedora 27, because it uses Wayland, and Gnome doesn't implement those under Wayland.

**Comment by nathan-osman** [comment]
Does it support AppIndicators? Does it require a third-party plugin? How do other "headless" applications handle this in Gnome?

**Comment by techtonik** [comment]
https://bugzilla.gnome.org/show_bug.cgi?id=758677 I am not sure what notifications do they speak about.

**Comment by techtonik** [comment]
https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome/ and this is official explanation.

**Comment by hedayat** [comment]
No, gnome is completely deprecating status icons. It'll disappear from Xorg too, since it won't exist in GTK 4 at all. 
Notifications are the ones that appear at the top of the screen, and if they are persistent, they'll reside in the box which is opened when you click at the center of the top bar (beside the calendar). However, some apps, e.g. music players, can also add a box there to control music playback. 

So, I see 2 solutions: 1. Implementing support for adding nitroshare to that area, like what music players do. 
2. Implementing a gnome shell extension exclusively for nitroshare, so that it is integrated in the top bar. 
I'd suggest the first option. 

Hey, actually the icon doesn't even appear in Fedora 27 using Xorg already!
